Pre-render item and profile pages before data is loaded

alse opened the pull request:
Continuation of #190

### Tests
- Profile and item pages are loading successfully on testnet

psiemens reviewed (commented):
My only concern with this approach [after seeing this](https://stackoverflow.com/a/66162437/1745785) is that I think it might prevent server-side prerendering. There are still parts of the page that can be rendered without `query` available and it'd be good to render as much as possible on the server.

Is there a way to optimize this?

alse commented:
@psiemens sure, I just updated it to pre-render the entire page and hide components that require data to be loaded. I also updated some of the swr queries to only run when they have the required data from params

10thfloor commented:
Hey @alse, will need these conflicts resolved before we can merge. I attempted, but could not achieve a working result.
